[TestChasteCitation fails](https://github.com/Chaste/Chaste/actions/runs/13272007657/job/37053404422) with PETSc 3.17.5

```
***** TestCitations.hpp *****


Entering TestChasteCitation
Testing Chaste implementation
Testing PETSc implementation

/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 4 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "  Author = {Satish Balay and Shrirang Abhyankar and Mark~F. Adams and Steven Benson and Jed Brown"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "  Author = {Satish Balay and Shrirang Abhyankar and Mark F. Adams and Jed Brown "
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 5 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Peter Brune and Kris Buschelman and Emil Constantinescu and Lisandro Dalcin and Alp Dener"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "            and Peter Brune and Kris Buschelman and Lisandro Dalcin and"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 6 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Victor Eijkhout and William~D. Gropp and V\'{a}clav Hapla and Tobin Isaac and Pierre Jolivet"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "            Victor Eijkhout and William D. Gropp and Dmitry Karpeyev and"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 7 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Dmitry Karpeev and Dinesh Kaushik and Matthew~G. Knepley and Fande Kong and Scott Kruger"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "            Dinesh Kaushik and Matthew G. Knepley and Dave A. May and Lois Curfman McInnes"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 8 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Dave~A. May and Lois Curfman McInnes and Richard Tran Mills and Lawrence Mitchell and Todd Munson"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "            and Richard Tran Mills and Todd Munson and Karl Rupp and Patrick Sanan"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 9 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Jose~E. Roman and Karl Rupp and Patrick Sanan and Jason Sarich and Barry~F. Smith"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "            and Barry F. Smith and Stefano Zampini and Hong Zhang and Hong Zhang},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 10 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "    and Stefano Zampini and Hong Zhang and Hong Zhang and Junchao Zhang},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "  Title = {{PETS}c Users Manual},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 11 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "  Title = {{PETSc/TAO} Users Manual},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "  Number = {ANL-95/11 - Revision 3.11},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 12 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "  Number = {ANL-21/39 - Revision 3.17},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "  Institution = {Argonne National Laboratory},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:255: Trace: "Line 13 differs in files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:256: Trace: "  Institution = {Argonne National Laboratory},"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:257: Trace: "  Year = {2019}"
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:268: Error: Expected (failures == 0u), found (34 != 0)
/__w/Chaste/Chaste/global/src/fortests/FileComparison.hpp:273: Trace: "Files /__w/Chaste/Chaste/chaste-test-dir/TestCitations/petsc_citations.txt and /__w/Chaste/Chaste/global/test/data/citations-2019.txt differ."
Failed

Failed 1 of 1 test
```

--- a/.github/workflows/portability.yml
+++ b/.github/workflows/portability.yml
@@ -19,15 +19,21 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - image: jammy.portability-dev
+          - image: portability-01
             extra-build-args: ""
-          - image: jammy.portability-dev
+          - image: portability-01
             extra-build-args: "-DChaste_USE_CVODE=OFF -DChaste_USE_VTK=OFF"
-          - image: focal.portability-system
+          - image: portability-focal
             extra-build-args: ""
-          - image: jammy.portability-system
+          - image: portability-jammy
             extra-build-args: ""
-          - image: noble.portability-system
+          - image: portability-lunar
+            extra-build-args: ""
+          - image: portability-mantic
+            extra-build-args: ""
+          - image: portability-noble
+            extra-build-args: ""
+          - image: portability-oracular
             extra-build-args: ""
 
     runs-on: ubuntu-latest

--- a/.github/workflows/portability.yml
+++ b/.github/workflows/portability.yml
@@ -23,6 +23,8 @@ jobs:
             extra-build-args: ""
           - image: portability-01
             extra-build-args: "-DChaste_USE_CVODE=OFF -DChaste_USE_VTK=OFF"
+          - image: portability-03
+            extra-build-args: ""
           - image: portability-focal
             extra-build-args: ""
           - image: portability-jammy

--- a/global/test/TestCitations.hpp
+++ b/global/test/TestCitations.hpp
@@ -129,6 +129,8 @@ public:
             FileFinder reference_citations("global/test/data/citations-2023.txt", RelativeTo::ChasteSourceRoot);
 #elif PETSC_VERSION_GE(3, 18, 0) //PETSc 3.18.0 onwards
             FileFinder reference_citations("global/test/data/citations-2022.txt", RelativeTo::ChasteSourceRoot);
+#elif PETSC_VERSION_GE(3, 17, 0) //PETSc 3.17.0 onwards
+            FileFinder reference_citations("global/test/data/citations-2022_0.txt", RelativeTo::ChasteSourceRoot);
 #elif PETSC_VERSION_GE(3, 11, 2) //PETSc 3.11.2 onwards
             FileFinder reference_citations("global/test/data/citations-2019.txt", RelativeTo::ChasteSourceRoot);
 #elif (PETSC_VERSION_MAJOR == 3 && (PETSC_VERSION_MINOR == 9 || PETSC_VERSION_MINOR == 10 || PETSC_VERSION_MINOR == 11)) //PETSc 3.9, 3.10, 3.11.1

--- a/global/test/data/citations-2022_0.txt
+++ b/global/test/data/citations-2022_0.txt
@@ -1,0 +1,37 @@
+If you publish results based on this computation please cite the following:
+===========================================================================
+@TechReport{petsc-user-ref,
+  Author = {Satish Balay and Shrirang Abhyankar and Mark~F. Adams and Steven Benson and Jed Brown
+    and Peter Brune and Kris Buschelman and Emil Constantinescu and Lisandro Dalcin and Alp Dener
+    and Victor Eijkhout and William~D. Gropp and V'{a}clav Hapla and Tobin Isaac and Pierre Jolivet
+    and Dmitry Karpeev and Dinesh Kaushik and Matthew~G. Knepley and Fande Kong and Scott Kruger
+    and Dave~A. May and Lois Curfman McInnes and Richard Tran Mills and Lawrence Mitchell and Todd Munson
+    and Jose~E. Roman and Karl Rupp and Patrick Sanan and Jason Sarich and Barry~F. Smith
+    and Stefano Zampini and Hong Zhang and Hong Zhang and Junchao Zhang},
+  Title = {{PETSc/TAO} Users Manual},
+  Number = {ANL-21/39 - Revision 3.17},
+  Institution = {Argonne National Laboratory},
+  Year = {2022}
+}
+@InProceedings{petsc-efficient,
+  Author = {Satish Balay and William D. Gropp and Lois Curfman McInnes and Barry F. Smith},
+  Title = {Efficient Management of Parallelism in Object Oriented Numerical Software Libraries},
+  Booktitle = {Modern Software Tools in Scientific Computing},
+  Editor = {E. Arge and A. M. Bruaset and H. P. Langtangen},
+  Pages = {163--202},
+  Publisher = {Birkh{\"{a}}user Press},
+  Year = {1997}
+}
+@article{Cooper2020Chaste,
+  doi = {10.21105/joss.01848},
+  url = {https://doi.org/10.21105/joss.01848},
+  year = {2020},
+  publisher = {The Open Journal},
+  volume = {5},
+  number = {47},
+  pages = {1848},
+  author = {Fergus R. Cooper and Ruth E. Baker and Miguel O. Bernabeu and Rafel Bordas and Louise Bowler and Alfonso Bueno-Orovio and Helen M. Byrne and Valentina Carapella and Louie Cardone-Noott and Jonathan Cooper and Sara Dutta and Benjamin D. Evans and Alexander G. Fletcher and James A. Grogan and Wenxian Guo and Daniel G. Harvey and Maurice Hendrix and David Kay and Jochen Kursawe and Philip K. Maini and Beth McMillan and Gary R. Mirams and James M. Osborne and Pras Pathmanathan and Joe M. Pitt-Francis and Martin Robinson and Blanca Rodriguez and Raymond J. Spiteri and David J. Gavaghan},
+  title = {Chaste: Cancer, Heart and Soft Tissue Environment},
+  journal = {Journal of Open Source Software}
+}
+===========================================================================

